bin/Grid_Bathy/hycom_grid.py

```diff
@@ -7,7 +7,7 @@
 import abfile.abfile as abf
 import modeltools.cice.io
 import numpy
-from mpl_toolkits.basemap import Basemap
+#from mpl_toolkits.basemap import Basemap
```

Comment:
simply remove anything related to basemap

Reply:
done